Makefile

```diff
@@ -303,7 +303,7 @@ catalog-push: ## Push a catalog image.
 
 .PHONY: build-push-iib
 build-push-iib: ## Build and push all the three images to quay (docker, bundle, and catalog).
-	docker-build bundle bundle-build container-push 
+	make docker-build bundle bundle-build container-push 
```

Comment:
Any reason to not just let `build-push-iib` depend on these targets? Same for the other one.
And I think `container-build container-push` is shorter and does the same.

Reply:
> Any reason to not just let `build-push-iib` depend on these targets? Same for the other one.

Previously when it was depending on these targets the after PR job failed, see for [#12 ](https://github.com/medik8s/node-maintenance-operator/runs/6666333343?check_suite_focus=true) and [#38](https://github.com/medik8s/node-maintenance-operator/runs/6687523429?check_suite_focus=true).
I am not sure I understand your suggestion. Do you suggest to add `make` in each of these targets (e.g., `docker-build, bundle-build, docker-push bundle-push catalog-build catalog-push`) instead of two `make` as in this PR?

> And I think `container-build container-push` is shorter and does the same.

IIUC `container-build` is additionally running `make test-no-verify` and `make bundle` on a new container, and we currently use it only in CI. Do you think it is better to add this logic?

Reply:
what I mean is use this, no need to call make at all:
```
build-push-iib: container-build container-push ## Build and push all the ...
```
and while writing this, I think `build-push-iib` isn't a good name:
- the target does more than building and pushing the iib
- at other place we use `catalog`, and not `iib`

>  the after PR job failed

I think it failed because of the issue we are trying to solve here, missing `make`?

> IIUC container-build is additionally running make test-no-verify

where do you see that?
it runs a containerized make bundle, and then docker-build bundle-build. Same targets as you run here?

Reply:
>no need to call make at all

I am aware of that, I just didn't though it is better to avoid using make.

> and while writing this, I think build-push-iib isn't a good name

What do you think about `creating-catalog-image`?

> I think it failed because of the issue we are trying to solve here, missing make?

Sure, and right now `missing make` doesn't sound like a good name to the PR and comment.

> where do you see that?

It first runs `check` which in turn runs `$(DOCKER_GO) "make test-no-verify"` (see [here](https://github.com/medik8s/node-maintenance-operator/blob/main/Makefile#L319))

Reply:
I agree, and I have updated it accordingly 👍🏻